src/course-unit/course-xblock/IframeComponent.jsx

```diff
@@ -0,0 +1,35 @@
+import { forwardRef } from 'react';
+import PropTypes from 'prop-types';
+
+import { IFRAME_FEATURE_POLICY } from './constants';
+
+const IframeComponent = forwardRef(({ title, ...props }, ref) => (
```

Comment:
```suggestion
const IFrameComponent = forwardRef(({ title, ...props }, ref) => (
```

Reply:
Maybe let's call it `CourseIFrame`? Because everything that returns `JSX` is `Component`. I don't think `Component` should be in the name of any component.

Reply:
Renamed, thanks